opent2t.p.light/com.wink.lightbulb/js/manifest.xml

```diff
@@ -0,0 +1,14 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<manifest>
+  <!-- associated voice handler -->
+  <voice id="org.opent2t.sample.lightcontrol.cortana.vcd.superpopular" />
+  <!-- associated schemas -->
+  <schemas>
+    <schema id="opent2t.p.light" main="true" type="OCF" />
+    <schema id="oic.core" type="OCF" />
+    <schema id="oic.baseresource" type="OCF" />
+    <schema id="oic.r.switch.binary" type="OCF" />
+    <schema id="oic.r.light.dimming" type="OCF" />
+    <schema id="oic.r.colour.rgb" type="OCF" />
+  </schemas>
```

Comment:
Missing a reference to the new opent2t.platform schema, as well as the oic.oic-link schema, and oic.r.colour.chroma.